### DISCUSSION TITLE
5.0.1 Released

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#3591] [General]
Merry Christmas!

https://github.com/oqtane/oqtane.framework/releases/tag/v5.0.1

**Comment by leigh-pointer** [comment]
> Merry Christmas!

And to you and family.
It has been a good year!!

**Reply by sbwalker** [reply]
15 releases, ,migration from .NET 6 to 7 to 8, creation of Oqtane Labs, launch of Oqtane Marketplace, and so much more... a very good year!

**Reply by leigh-pointer** [reply]
What is even more fantastic is, the direction we are moving in; as a product and a team 💪

**Reply by thabaum** [reply]
Hope you all are enjoying the holiday season with a Merry Christmas and have a Happy New Year!

It takes teamwork to make the dream work!

BIG THANKS to everyone helping by being a part of community here with everything happening getting Oqtane Framework where it is with the pedal to the metal to get where it is going!  This is a fun and exciting project to be a part of!

Cheers!

**Comment by leigh-pointer** [comment]
Language packs 5.0.1 NL (Nederlands) and GB (British) are also ready for download.

**Comment by thabaum** [comment]
> 15 releases, ,migration from .NET 6 to 7 to 8, creation of Oqtane Labs, launch of Oqtane Marketplace, and so much more... a very good year!

Let's not forget to mention the start of developer meetings, which we have one one tomorrow 12pm EST as it is the first Wednesday of the month!

Hope everyone contributing can make it and more join up in 2024 helping fuel Oqtane Framework development with .NET 8!  

Anyone who contributed I believe can get included in the list to join the Team meeting.  Just fill out the contact form on the Oqtane website and request to be included on the developer meeting contact list as a reminder to anyone that has not been able to join.  Do this as soon as you can prior to any meeting to get the connection information sent to your email.
